cmd/cri-containerd/options/options.go

```diff
@@ -18,38 +18,56 @@ package options
 
 import (
 	"flag"
+	"os"
 
+	"github.com/BurntSushi/toml"
 	"github.com/containerd/containerd"
 	"github.com/spf13/pflag"
 )
 
-// CRIContainerdOptions contains cri-containerd command line options.
-type CRIContainerdOptions struct {
+const configFilePathArgName = "config-file-path"
+
+// ContainerdConfig contains config related to containerd
+type ContainerdConfig struct {
+	// ContainerdSnapshotter is the snapshotter used by containerd.
+	ContainerdSnapshotter string `toml:"containerdsnapshotter"`
+	// ContainerdEndpoint is the containerd endpoint path.
+	ContainerdEndpoint string `toml:"containerdendpoint"`
+}
+
+// Config contains cri-containerd toml config
+type Config struct {
+	// contains config related to containerd
+	ContainerdConfig
 	// SocketPath is the path to the socket which cri-containerd serves on.
-	SocketPath string
+	SocketPath string `toml:"socketpath"`
```

Comment:
Use underscore for multiple words? Both containerd and cri-o are doing so.

This should be `socket_path`. And also for the following ones.